avalancheup-aws/src/apply/mod.rs

```diff
@@ -2726,14 +2738,32 @@ default-spec --log-level=info --funded-keys={funded_keys} --region={region} --up
             )),
             ResetColor
         )?;
-        let asg_tmpl = aws_dev_machine::artifacts::asg_ubuntu_yaml().unwrap();
+        let asg_tmpl =
+            avalanche_ops::dev_machines::dev_machine_artifacts::asg_ubuntu_yaml().unwrap();
 
         let mut cfn_params = Vec::new();
         for (k, v) in regional_common_dev_machine_asg_params.iter() {
             log::info!("dev-machine CFN parameter: '{k}'='{v}'");
             cfn_params.push(build_param(k, v));
         }
 
+        // Add the security group ids to the dev machine ASG
+        cfn_params.push(build_param(
+            "NetworkInterfaces.0.Groups.0",
```

Comment:
Does this work? Just using `DevSecurityGroupId` parameter wouldn't work?

Reply:
I think you're right. We already defined the `SecurityGroupId` so updating the dev one should be fine in the `!Join`. Updated. I think this is ready